freetype/raw.py

```diff
@@ -12,23 +12,35 @@
 import platform
 from ctypes import *
 import ctypes.util
+import pkg_resources
 
 from freetype.ft_types import *
 from freetype.ft_enums import *
 from freetype.ft_errors import *
 from freetype.ft_structs import *
 
-# on windows all ctypes does when checking for the library
-# is to append .dll to the end and look for an exact match
-# within any entry in PATH.
-filename = ctypes.util.find_library('freetype')
+system = platform.system()
+if system == 'Windows':
+    library_name = 'freetype.dll'
+elif system == 'Darwin':
+    library_name = 'libfreetype.6.dylib'
+else:
+    library_name = 'libfreetype.so.6'
 
-if filename is None:
-    if platform.system() == 'Windows':
-        # Check current working directory for dll as ctypes fails to do so
-        filename = os.path.join(os.path.realpath('.'), 'freetype.dll')
-    else:
-        filename = 'libfreetype.so.6'
+filename = pkg_resources.resource_filename('freetype', library_name)
```

Comment:
using `pkg_resources` adds a runtime dependency on setuptools -- which nowadays is preinstalled on most python distributions alongside pip, but technically is not part of the python standard library.
The old-school approach to get a resource filename relative to `__file__` works fine, unless you want to get a filename of a data file from a inside a zipped python package (like an .egg). But since we are including a precompiled native library, we have to set `zip=False` in setup.py in any case, otherwise we can't load it. So using pkg_resources has no concrete advantage IMO over `os.path.dirname(__file__, library_name)`